src/main/java/org/folio/notes/service/impl/TypeServiceImpl.java

```diff
@@ -0,0 +1,105 @@
+package org.folio.notes.service.impl;
+
+import java.time.OffsetDateTime;
+import java.util.List;
+import java.util.UUID;
+import java.util.stream.Collectors;
+import javax.persistence.EntityNotFoundException;
+
+import lombok.RequiredArgsConstructor;
+import lombok.extern.slf4j.Slf4j;
+import org.springframework.beans.factory.annotation.Value;
+import org.springframework.stereotype.Service;
+
+import org.folio.notes.domain.dto.Metadata;
+import org.folio.notes.domain.dto.NoteType;
+import org.folio.notes.domain.dto.NoteTypeCollection;
+import org.folio.notes.domain.entity.TypeEntity;
+import org.folio.notes.domain.repository.TypeRepository;
+import org.folio.notes.exception.TypesLimitReached;
+import org.folio.notes.mapper.TypeMapper;
+import org.folio.notes.service.TypeService;
+import org.folio.spring.data.OffsetRequest;
+
+@Slf4j
+@Service
+@RequiredArgsConstructor
+public class TypeServiceImpl implements TypeService {
+
+  @Value("${folio.notes.types.default.name}")
+  private String defaultNoteTypeName;
+
+  @Value("${folio.notes.types.number.limit.default}")
```

Comment:
```suggestion
  @Value("${folio.notes.types.amount.limit.default}")
```
Maybe this will be better? What do you think?

Reply:
What about this? It will be similar with default name
`${folio.notes.types.default.limit}`